hdr_schemata/models/GWDM/v2_0/Revision.py

```diff
@@ -13,7 +14,7 @@ class Revision(BaseModel):
         **an.version.__dict__,
     )
 
-    url: Url = Field(
+    url: Optional[Url] = Field(
```

Comment:
This brings the GWDM into alignment with the HDR schema where the Url is optional.